bundles/org.openhab.binding.siemensrds/README.md

```diff
@@ -0,0 +1,126 @@
+
+## Channels for RDS Smart Thermostat
+
+The RDS Smart Thermostat supports several channels as shown below. 
+
+| Channel                  | Data Type | Description                                                                 |
+|--------------------------|-----------|-----------------------------------------------------------------------------|
+| roomTemperature          | Number    | Actual Room Temperature                                                     |
+| targetTemperature        | Number    | Target temperature setting for the room                                     |
+| thermostatOutputState    | String    | Status of whether the thermostat is Heating, Cooling, or Neither            |
+| roomHumidity	           | Number    | Actual Room Humidity                                                        |
+| roomAirQuality           | String    | Actual Room Air Quality (Poor..Good)                                        |
+| outsideTemperature       | Number    | Actual Outside temperature                                                  |
+| greenLeafScore           | String    | Green Leaf Mode / Energy saving level (Poor..Excellent)                     |
+| occupancyModePresent     | Switch    | The Thermostat is in the Present Occupancy Mode (Off=Absent, On=Present)    |
+| thermostatAutoMode       | Switch    | The Thermostat is in Automatic Mode (Off=Manual, On=Automatic)              |
+| hotWaterAutoMode         | Switch    | The Domestic Water Heating is in Automatic Mode (Off=Manual, On=Automatic)  |
+| hotWaterSwitchState      | Switch    | The On/Off state of the domestic water heating                              |
+
+## Full Example
+
+### `demo.things` File
+
+As a general rule it is **recommended** to create the Thing for Climatix IC Account using the PaperUI; however if you wish to do so, you *MAY* create the account thing by means of a `.things` file as follows..
+
+```
+Bridge siemensrds:climatixic:mybridgename "Climatix IC Account" [ userEmail="email@example.com", userPassword="secret", apiKey="32-character-code-provided-by-siemens", pollingInterval=60 ]
+}
+```
+
+Note: it is **strongly** recommended **NOT** to create RDS Smart Thermostat Things through manual configuration files. The reason is that to configure an RDS Smart Thermostat Thing requires advance knowledge of the "Plant Id" which is a unique code used to identify a specific thermostat device in the Siemens Climatix IC cloud server account. The PaperUI automatic Discovery service (see above) discovers the unique "Plant Id" codes during the discovery process. But if you wanted to create an RDS Smart Thermostat by means of a manual configuration file you would not know its Plant Id. *Nevertheless, for completeness sake, the following shows an example of a manual configuration file, assuming that you had determined the Plant Ids by previously using the auto discovery service*..        
```

Comment:
Please have a line break after every sentence.
Don't do recommendations here. As discussed, both ways are equally valid. Just point out that a plantId is required and that this can simplest be obtained through auto-discovery.

Reply:
Ok. Done